dockerfiles/hpcc/templates/eclagent.yaml

```diff
@@ -29,7 +29,7 @@ spec:
 {{ include "hpcc.addVolumes" $ | indent 6 }}
 ---
 {{- include "hpcc.generateConfigMap" (dict "root" $ "component" "eclagent" "me" .) }}
-  jobspec.yaml: |
+  eclagent.yaml: |
```

Comment:
not wild about the name - eclagent-job or something perhaps?

Reply:
agree, will change these.

Reply:
however.. this is found via the component name at the moment.
so e.g. :
                runK8sJob("eclagent", wuid, wuid);

runK8Job uses componentName to find resources to set from the workunit in setResources.

Could be jobspec name, and component/type name..